x/ibc/core/24-host/keys.go

```diff
@@ -105,104 +128,108 @@ func KeyConnection(connectionID string) []byte {
 
 // ChannelPath defines the path under which channels are stored
 func ChannelPath(portID, channelID string) string {
-	return fmt.Sprintf("%s/", KeyChannelPrefix) + channelPath(portID, channelID)
+	return fmt.Sprintf("%s/%s", KeyChannelEndPrefix, channelPath(portID, channelID))
+}
+
+// ChannelKey returns the store key for a particular channel
+func KeyChannel(portID, channelID string) []byte {
```

Comment:
Why isn't this `ChannelKey`?

Reply:
good question

Reply:
[fixed](https://github.com/cosmos/cosmos-sdk/pull/7873/commits/817743e610b5c3717119faad3cafdbf8cc0fbcf2), forgot to commit before fixing merge conflicts